experimental/torch_xla2/torch_xla2/ops/jtorch.py

```diff
@@ -414,3 +424,43 @@ def linalg_tensorsolve(A, b, dims=None):
   if A.shape[:b.ndim] != b.shape:
     b = jnp.reshape(b, A.shape[:b.ndim])
   return jnp.linalg.tensorsolve(A, b, axes=dims)
+
+@register_function(torch.linspace)
+def linspace(start, end, steps=100, *, dtype=None, device=None, **kwargs):
+    env = kwargs.get("env")
+
+    if dtype is None:
+        dtype = torch.float32
+
+    jdtype = TORCH_TO_JAX_DTYPE.get(dtype)
```

Comment:
Can use this function instead https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/torch_xla2/ops/mappings.py#L88

Reply:
Got it